### ISSUE TITLE
Struggling with SONY BRAVIA KD43XE8077SU

### ISSUE THREAD

**Issue by EmmanuelLM** [aparraga/braviarc#18]
Using the latest version of BraviaRC on Home-Assistant (Hassio) version 0.55.2

Description of problem:
I am struggling to configure my TV above with Home-Assistant. Here is what I tried (and what happened):

added the “platform: braviatv” with the correct host IP; no bravia.conf file. Rebooted everything. The configuration module appears, enter 0000, a pin then appears on my TV, which I then enter in HA - the TV then says “configuration cancelled” and nothing else happens… note: I had removed all prior associated devices on the TV;
as above but created a bravia.conf with the right host, pin, mac (quick question: the pin is the pre-shared pin? and the mac is the tv mac, right? that is what I have assumed anyway). Now two things happen after I reboot everything:
2.1) if the TV is off whilst HA is rebooting: I see a Sony Bravia TV module and can see the on/off state, but none of the controls work;
2.2) if the TV is on whilst HA is rebooting: I cannot see a module in HA and the TV presents me with a pin to enter every 30 seconds or so (the prompt only stays on for 10 seconds or so), I have no configuration modules etc…
In terms of config, everything is up to date etc…

Basically, I am stumped, don’t really know what else to try, so any help is appreciated!

Expected:
Configuration panel shows, enter 0000, pin is displayed on TV, enter the pin from the TV in the panel, TV added as a component.

Problem-relevant configuration.yaml entries and steps to reproduce:

media_player:
  - platform: braviatv
    host: 192.168.1.234
Click on configure in the Bravia configuration panel
Enter 0000, Click confirm, a pin is displayed on the TV
Enter the pin from the TV in the panel, press 'Confirm' - TV says 'configuration cancelled', panel says wrong pin
